backend/services/stats_service.py

```diff
@@ -547,10 +546,14 @@ def get_task_stats(
                 TaskHistory.action_text,
                 func.DATE(TaskHistory.action_date).label("day"),
             )
+            .order_by(
+                TaskHistory.project_id,
+                TaskHistory.task_id,
+                TaskHistory.action_text,
+                TaskHistory.action_date,
+            )
```

Comment:
Same here.. If this is not related to dependency updates and needs change. Let's do this in another PR.

Reply:
Works for me. I'll check to make certain that it doesn't break tests first though.